docs/source/https.md

```diff
@@ -0,0 +1,53 @@
+# Enabling HTTPS
+
+In order to protect the data and passwords transfered over the public network, you should not run JupyterHub without enabling **HTTPS**.
+
+JupyterHub Traefik-Proxy supports **automatically** configuring HTTPS via [Let’s Encrypt](https://letsencrypt.org/docs/), or setting it
+up **manually** with your own key and certificate.
+
+## How-To HTTPS for JupyterHub managed Traefik-Proxy
+1. Via **Let'Encrypt**:
+    * Enable automatic https:
+    ```python
+    c.Proxy.traefik_auto_https=True
+    ```
+    * Configure a HTTPS port. This is where all the traffic coming through the HTTP entrypoint will be redirected to:
+    ```python
+    c.Proxy.traefik_https_port=443
+    ```
+    * Set the email address used for Let's Encrypt registration:
+    ```python
+    c.Proxy.traefik_letsencrypt_email="jovyan@jupyter.test"
```

Comment:
I suggest we leave this blank, and let it fail if it is and it is needed. That way, we don't risk having people using this code as it and sending requests with user accounts associated with Jupyter which is not associated with an actual human.

```suggestion
    c.Proxy.traefik_letsencrypt_email=""
```